example/spec/retryConfiguration/retryConfiguration.js

```diff
@@ -0,0 +1,100 @@
+const { buildAndExecuteWorkflow, LambdaStep } = require('@cumulus/integration-tests');
```

Comment:
I think this file needs to be retryConfigurationSpec, or the spec/support/jasmine.json needs to be updated so npm test will run this spec.

Reply:
Tests failing on my local run were just a deployment issue.  Looks good to me aside the above comments.